jkube-kit/build/service/docker/src/test/java/org/eclipse/jkube/kit/build/service/docker/helper/ImageNameFormatterTest.java

```diff
@@ -131,4 +223,8 @@ void format_whenPropertyInImageName_thenResolveProperty() {
         // Then
         assertThat(result).isEqualTo("registry.gitlab.com/myproject/myrepo/mycontainer:der12");
     }
+
+    private static void validImageName(String v) {
+        assertThatCode(() -> ImageName.validate(v)).doesNotThrowAnyException();
```

Comment:
Is this an assertion, an action, both? Can we write a specific test for this?
Can we move the assertion part to other scenarios?

Reply:
this seems to be an assertion. `ImageName.validate` creates a new ImageName object which throws IllegalArgumentException if invalid name is provided